### PR TITLE
Partially fixed #3756 - Calendar pop-ups now auto close after 500ms w…

### LINKED ISSUE #3756: Popup Studio and Calendar don't auto-close
#### Issue

When saving chaned labels in Studio, or when hovering over an event in the Calendar, a popup appears. This popup doesn't automatically disappear when expected.

#### Expected Behavior

Especially for Calendar events, I expect these popups to auto close when not hovering over the event anymore.

In Studio, this messages you get when saving changed labels, for example, used to be a message at the top of the screen that automatically disappear after a certain amount of time. Also, the screen isn't blocked, so you can continue working, so the work flow isn't interrupted.

#### Actual Behavior

Calendar event popups need a manual click on the close button to close.
Studio popups need a manual click on the close button to close, blocking the screen and all functions until you do so.

#### Steps to Reproduce

Calendar popups:

1. Create an event that appears on the Calendar
2. Go to the Calendar and hover over this event

Studio popups

1. Go to Labels of a module
2. Make a change and click save

#### Your Environment

* SuiteCRM Version used: 7.8.1 + SuiteP
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Firefox & Chrome
* Environment name and version (e.g. MySQL, PHP 7): Php 5.6 & 7.0
* Operating System and version (e.g Ubuntu 16.04): Centos 7

### PR DESCRIPTION
**Pull request by ebogaard**
…hen the mouse leaves the event or the popup itself

## Description

See issue: #3756 

## Motivation and Context

Now, pop-ups don't auto-close, cluttering the Calendar screen.

## How To Test This

Create an event that shows up on the Calendar. GO to the Calendar, hover your mose over the event and see if the pop-up auto-closes 500ms after mouseout.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [n.a.] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by samus-aran** [comment]
@ebogaard I've just updated the conflicts and we'll get the team to test this

**Comment by Mac-Rae** [comment]
Assessed +1

**Comment by samus-aran** [comment]
Restarting a Travis test as other env are passing.

File: jssource/src_files/modules/Calendar/Cal.js [hunk 1]
```diff
@@ -1099,6 +1098,10 @@ $($.fullCalendar).ready(function () {
             },
             show: {solo: true},
             hide: {event: false},
+							event: 'mouseleave',
```
Comment (mattlorimer) [inline jssource/src_files/modules/Calendar/Cal.js]: your code is indented out of line with the rest of the code in this section, in this case I would suggest the indents should be consistent with the rest of the code in the file.

**Comment by samus-aran** [comment]
Whoops, my fault! I was resolving conflicts via web editor <<; I'll update tomorrow morn.

**Comment by cameronblaikie** [comment]
Assessed :+1:

**Pull request merged**